sshfs.c

```diff
@@ -4173,6 +4205,7 @@ int main(int argc, char *argv[])
 	sshfs.truncate_workaround = 0;
 	sshfs.buflimit_workaround = 0;
 	sshfs.createmode_workaround = 0;
+	sshfs.readdir_workaround = 1;
```

Comment:
I think a workaround should always be disabled by default.

Reply:
See below